integration_tests/pages/refer/programmeHistoryDetails.ts

```diff
@@ -0,0 +1,71 @@
+import Page from '../page'
+
+export default class ProgrammeHistoryDetailsPage extends Page {
+  constructor() {
+    // Conditional radio buttons add an additional `aria-expanded` field,
+    // so ignore that rule on this page
+    super('Add Accredited Programme history', { accessibilityRules: { 'aria-allowed-attr': { enabled: false } } })
+  }
+
+  shouldContainOutcomeDetailTextArea() {
+    this.shouldContainTextArea('outcomeDetail', 'Can you provide additional detail? (if known)')
+  }
+
+  shouldContainOutcomeOptions() {
+    cy.get('[data-testid="outcome-options"]').within(() => {
+      this.shouldContainRadioItems([
+        { label: 'Complete', value: 'complete' },
+        { label: 'Incomplete', value: 'incomplete' },
+      ])
+    })
+  }
+
+  shouldContainSettingOptions() {
+    cy.get('[data-testid="setting-options"]').within(() => {
+      this.shouldContainRadioItems([
+        { label: 'Community', value: 'community' },
+        { label: 'Custody', value: 'custody' },
+      ])
+    })
+  }
+
+  shouldContainSourceTextArea() {
+    this.shouldContainTextArea('source', 'Provide the source')
+  }
+
+  shouldDisplayCommunityLocationInput() {
+    cy.get('[data-testid="community-setting-option"]').check()
+    cy.get(`input[id="communityLocation"]`).should('be.visible')
```

Comment:
Here and in the rest of the methods that follow: the backticks can be single quotes (there's no interpolation going on)

Reply:
Good spot